app/controllers/application_controller.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  before_action :authenticate_employee!
```

Comment:
usunąłeś autoryzacje employee w Application controller, i dodałeś ją w EmployeeController, jednakże KudosController dalej dziedziczy po ApplicationController, a więc teraz nie ma tam autentykacji employee

Reply:
powiem Ci tak, że pare minut po tym jak Ci wysłałem PRa to się kapnąłem, że tak właśnie jest i że prawdą jest to co napisaleś w komentarzu wyżej, że mozna oglądać kudosy bez zalogowania się do employeea i zmeniłem dziedziczenie w Kudosach :)